core/lib/zksync_core/src/lib.rs

```diff
@@ -630,6 +630,25 @@ pub async fn initialize_components(
                     Arc::new(ValidiumModeL1BatchCommitDataGenerator {})
                 }
             };
+
+        let args = CallFunctionArgs::new("getPubdataPriceMode", ()).for_contract(
```

Comment:
Also, probably it would make sense to encapsulate this logic to a separate function. For now I think we can keep it in the same file.

Reply:
Done in [d58e1ac](https://github.com/matter-labs/zksync-era/pull/1240/commits/d58e1acd5b3d4d8393fa8395f038d8ec731f10ab)